task/icr-scan/0.1/icr-scan.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: icr-scan
+spec:
+  description: >-
+    The following task can be used to perform analysis on the source code
+    and send the results to the iCR server specified by the parameter Server-URL
+  params:
+
+    - name: iCR-License
+      description: The license required to use the iCR to scan the source code.
+
+    - name: Git-Revision
+      description: Branch of the Git Repository
+      default: master
+
+    - name: Server-URL
+      description: The URL of the Server where data will be sent
+
+    - name: iCR-Username
+      description: The Username for the iCR Server
+      default: admin
+
+    - name: iCR-Password
+      description: The Password for the iCR Server 
+      default: admin   
+
+  steps:
+    - name: icr-scan
+      image: docker.io/munahaf/redhat-tekton:latest
+
+      resources:
+        requests:
+          memory: 8Gi
+          cpu: 1000m
+        limits:
+          memory: 8Gi
+          cpu: 2500m
```

Comment:
The task is very resource intensive and as of now, we cannot bring this down.

Reply:
The task is very resource-intensive and we cannot bring this down as of now.

Reply:
since this is resource intensive and can lead to high costs, can you please add a disclaimer in README in that case, so that users of this task are aware well in advance

Reply:
@vinamra28 I have added a Disclaimer in the Read Me, please check the latest commit. Thanks